### PR TITLE
03 bash hw 2

### PR DESCRIPTION
**Pull request by Happybolt**
Script to search for files with the same md5  checksum

### CODE REVIEW

File: 03-Bash/find_dup_files/find_dup_files.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+path=$(pwd)
+
+if [[ $1 = "-h" || $1 = "--help" ]]; then
+    echo "Script to search for files with the same md5  checksum"
+    echo "Usage: sript [OPTION] or [PATH]"
```
Comment (yekovalyov) [inline 03-Bash/find_dup_files/find_dup_files.sh]: Change sript to find_dup_files.sh or to $0

Comment (Happybolt) [reply 03-Bash/find_dup_files/find_dup_files.sh]: Done

File: 03-Bash/find_dup_files/find_dup_files.sh [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    exit 0
+fi
+
+if [ -n "$1" ]; then
+    path="$1"
+fi
+
+echo "$path"
+
+touch list_files
```
Comment (yekovalyov) [inline 03-Bash/find_dup_files/find_dup_files.sh]: The name for temporary file is not unique for each program run.

Comment (Happybolt) [reply 03-Bash/find_dup_files/find_dup_files.sh]: Fixed. Add use mktemp

**Pull request merged**